### ISSUE TITLE
Security related typo: `--no-ssl` in `docker/README.adoc` while might meant `--no-sso`

### ISSUE THREAD

**Issue by Benjamin-Loison** [8go/matrix-commander#136]
https://github.com/8go/matrix-commander/issues/80#issuecomment-1159166706 mentions the security related typo. [It's indeed a typo](https://github.com/8go/matrix-commander/blob/dbf2829ac942c360834195e12b90b1573ba55d00/docker/README.adoc?plain=1#L49), as [this mention](https://github.com/8go/matrix-commander/blob/dbf2829ac942c360834195e12b90b1573ba55d00/docker/README.adoc?plain=1#L15-L16) makes us think.

However when running the command with `--no-sso`, we have the error:

```
usage: matrix-commander [--usage] SNIP
matrix-commander: error: unrecognized arguments: --no-sso
```

**Comment by 8go** [comment]
Ok, the page is out of date. Thank you @Benjamin-Loison for bringing this up and identifying the incorrect documentation. 

The software has been changed in the past and the correct usage is now `matrix-commander --login password`.

I will fix the README file to reflect this correct usage.

**Comment by 8go** [comment]
Fixed, see commit 026450f111d8600fe4cec29ba62ad17112a0d545
